manifests/h/Huawei/WeMeeting/1.2.0.5/Huawei.WeMeeting.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Huawei.WeMeeting
+PackageVersion: 1.2.0.5
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine

```

```suggestion
  Scope: user
```
This application installs to `C:\Users\WDAGUtilityAccount\AppData\Roaming\WeMeeting` which is per user instead of machine-wide.

`Computer\HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\WeMeeting`